### PR TITLE
Use Gravatar for avatars

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

**Comment by gela-d** [comment]
@tormozz48 @tavriaforever @tadatuta

File: src/model/nodes/person.js [hunk 1]
```diff
@@ -36,6 +37,19 @@ PersonNode.prototype.setTitle = function (person) {
 };
 
 /**
+ * Gets email for node
+ * @param {Object} person - object data for person
+ * @returns {PersonNode}
+ */
+PersonNode.prototype.getEmail = function (person) {
+    this.email = utility.getLanguages().reduce(function (prev, lang) {
+        prev[lang] = util.format('%s', person[lang]['email']);
```
Comment (tadatuta) [inline src/model/nodes/person.js]: what is the reason to use `util.format()` here?

as for codestyle, I prefer to write property lookup in dot notation (`person[lang].email`).

Comment (gela-d) [inline src/model/nodes/person.js]: @tadatuta 
my falt, update PR

File: src/model/nodes/person.js [hunk 2]
```diff
@@ -36,6 +37,19 @@ PersonNode.prototype.setTitle = function (person) {
 };
 
 /**
+ * Gets email for node
+ * @param {Object} person - object data for person
+ * @returns {PersonNode}
+ */
+PersonNode.prototype.getEmail = function (person) {
```
Comment (tormozz48) [inline src/model/nodes/person.js]: setEmail

**Pull request merged**